service/notifier.go

```diff
@@ -250,3 +262,29 @@ func (p *notifier) decodeSearchAttrBinary(bytes []byte, key string) interface{}
 	}
 	return val
 }
+
+func (p *notifier) sendMessageToWebhook(notification *Notification, webhook *config.Webhook) error {
+	var jsonStr = []byte(fmt.Sprintf("%v", notification))
+	req, err := http.NewRequest("POST", webhook.URL.String(), bytes.NewBuffer(jsonStr))
+	if err != nil {
+		return err
+	}
+	req.Header.Set("X-Custom-Header", "myvalue")
+	req.Header.Set("Content-Type", "application/json")
+
+	// TODO: setup retry logic using webhook config
+	client := &http.Client{}
```

Comment:
maybe move this client to notifier so that we don't have to create a client every time.